loom/common/platform/platformAndroidJni.cpp

```diff
@@ -294,7 +294,6 @@ const char *LoomJni::getWritablePath()
         "()Ljava/lang/String;"))
     {
         jstring str = (jstring)t.getEnv()->CallStaticObjectMethod(t.classID, t.methodID);
-        t.getEnv()->DeleteLocalRef(t.classID);
```

Comment:
Gone. No real object instance is involved here. This was crashing for that exact reason - no reference to delete.